edx_sga/sga.py

```diff
@@ -36,13 +36,18 @@
 
 
 log = logging.getLogger(__name__)
-
+BLOCK_SIZE = 8 * 1024
 
 def reify(meth):
     """
     Property which caches value so it is only computed once.
+    Keyword arguments:
+    inst
```

Comment:
Actually the keyword argument of this function is `meth`. This would be more useful: "meth: method to reify".